addons/apecloud-mysql/apecloud-mysql-addon.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions.kubeblocks.io/v1alpha1
+kind: Addon
+metadata:
+  name: apecloud-mysql
+  labels:
+    {{- include "kubeblocks.labels" . | nindent 4 }}
+    "kubeblocks.io/provider": apecloud
+  {{- if .Values.keepAddons }}
+  annotations:
+    helm.sh/resource-policy: keep
+  {{- end }}
+spec:
+  description: 'ApeCloud MySQL is a database that is compatible with MySQL syntax and achieves high availability
+    through the utilization of the RAFT consensus protocol.'
+
+  type: Helm
+
+  helm:
+    {{- include "kubeblocks.addonChartLocationURL" ( dict "name" "apecloud-mysql" "version" (default .Chart.Version .Values.versionOverride) "values" .Values) | indent 4 }}
```

Comment:
Do we need to make corresponding changes here when dependencies on 'KubeBlock' are removed from the addon? Alternatively, should we add a readme in the subfolder like `\addons` or `\tools` to help third-party users understand the meanings of these fields

Reply:
Yes, we need. This PR only move addon CRs to this repo, more works will be done in the next PRs.

Reply:
> Yes, we need. This PR only move addon CRs to this repo, more works will be done in the next PRs.

Get it!